### DISCUSSION TITLE
Will there be some kind of faucet to new comers to invest into Chia?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#2844] [Ideas]
As investment barrier for Chia is getting higher and higher day by day, is there a faucet mechanism being built to reward new individual farmers as an incentive to stay interested?  I believe that this would avoid the scenario that there is zero CHX winning for maybe months if not years to come after significant personal investment.

Typical household won't have the need to get so many TB of HDD space for everyday use. So although the original intent for the space farming is great: using existing spare HDD spaces. However, things seem not to go that way, as individuals might be buying up spaces they may never need for their entire life. Statistically, yes, overall, someone somewhere will win some CHX. However, that's not a good investment strategy in general.  You would rather want small but more steady rewards, instead of sporadic larger rewards with long waiting time that create huge uncertainty.

**Comment by pluto2021** [comment]
As an example, certain incentives for each Plot one would contribute to the netspace.

**Comment by cccat6** [comment]
What you are asking is called "pool".

**Reply by pluto2021** [reply]
Similar, but not exactly. I was talking about a fixed amount of CHX (say a fraction) for each Plot added.
